pom.xml

```diff
@@ -18,16 +18,23 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Do you need both JUnit and TestNG?

Reply:
> Do you need both JUnit and TestNG?
Yep, since other tests in current project (e.g. plates, quicksort) were written by using TestNG.